reproducer-clean.yml

```diff
@@ -25,6 +25,24 @@
         name: rhol_crc
         tasks_from: cleanup.yml
 
+    - name: Remove remote selected data directories
+      tags:
+        - deepscrub
+      ansible.builtin.file:
+        path: "{{ cifmw_reproducer_basedir }}/{{ item }}"
+        state: absent
+      loop:
+        - reproducer-inventory
+        - reproducer-network-env
+
+    - name: Remove local selected data directories
```

Comment:
just a comment on the name, this one is not selecting directories more like "nuke entire ci-framework-data directory"

Reply:
yeah, I'll update the directory to only a selected one related to reproducer.